package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "linter": "./node_modules/eslint/bin/eslint.js './**/*.js'",
-    "start": "node index.js",
+    "start": "node --require babel-register index.js",
```

Comment:
@Danielshow will this script run babel? I don't think it will transpile the code.

Reply:
Also, it will be tidy if the transpiled code is saved in a build folder and node starts the transpiled code.